templates/deno/src/role.ts.twig

```diff
@@ -0,0 +1,28 @@
+export class Role {
+    public static any(): string {
+        return 'any'
+    }
+
+    public static user(id: string): string {
+        return 'user:\(id)'
+    }
+    
+    public static users(): string {
+        return 'users'
+    }
+    
+    public static guests(): string {
+        return 'guests'
+    }
+    
+    public static team(id: string, role: string = ''): string {
+        if(role == '') {
```

Comment:
Let's use the language builtin function for checking for an empty string.. 
If that is not available, we should use strict equality checks using `===` if available

Reply:
done 👍